packages/contractkit/src/utils/tx-signing.ts

```diff
@@ -20,38 +27,144 @@ function getPrivateKeyWithout0xPrefix(privateKey: string) {
   return privateKey.toLowerCase().startsWith('0x') ? privateKey.substring(2) : privateKey
 }
 
-export class CeloProvider extends PrivateKeyWalletSubprovider {
+function isNullOrUndefined(value: any): boolean {
```

Comment:
style: unnecessary. In js, we use `x == null` to test this. In fact, that's the only valid use of `==` vs `===`.